web/src/beta/features/Dashboard/ContentsContainer/Members/ListItem.tsx

```diff
@@ -5,28 +5,24 @@ import { useT } from "@reearth/services/i18n";
 import { styled } from "@reearth/services/theme";
 import { FC, useCallback } from "react";
 
+import { PermissionService } from "./PermissionService";
+
 const ListItem: FC<{
   member: TeamMember;
   setUpdateRoleModalVisible: (visible: boolean) => void;
   setSelectedMember: (member: TeamMember) => void;
   setDeleteMemerModalVisible: (visible: boolean) => void;
-  meRole: string | undefined;
-  meId: string | undefined;
+  meRole: Role | undefined;
+  isLast: boolean;
```

Comment:
`isLast` can't present the meaning fully i think. `isLastOwner` might be better.

Reply:
Better add some description as comment on code explain its usage.